### PR TITLE
[#171705742] Add Startup IDPS metadata setting procedure into README

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +249,23 @@ Kubernetes secrets:
...
+
+The certificate and the secret key could be provided to the backend with two ENV variables:
+- `SAML_KEY`
+- `SAML_CERT`
+
+if these variables are set `cert.pem` and `key.pem` are ignored.
+
+### SPID Identity Providers Metadata
+
+When backend starts, SPID login is configured with Identity Providers metadata fetched from remote URL. URLs are provided by the following Environment variables: `CIE_METADATA_URL` (CIE login) `IDP_METADATA_URL` (SPID login) and `SPID_TESTENV_URL` (SPID test environment). Before that the backend become ready a time consuming operation must be completed (fetching remote metadata).
```
Comment (gunzip) [inline README.md]: can you add a table with the default values from both test (local development) and prod environments ?

Comment (BurnedMarshal) [reply README.md]: Added into 344c47b

File: README.md [hunk 2]
```diff
@@ -249,6 +251,31 @@ Kubernetes secrets:
...
+
+The default values are explained into the following table
+
+| ENV                                    | stage                                                                             | production                                                       |
+|----------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| `IDP_METADATA_URL`                       | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml                  | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml |
+| `CIE_METADATA_URL`                       | https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth                    | https://idserver.servizicie.interno.gov.it/idp/shibboleth        |
+| `SPID_TESTENV_URL`                       | https://spid-testenv.dev.io.italia.it                                             | -                                                                |
```
Comment (gunzip) [inline README.md]: it looks like we won't have a staging environment deployed (only production)
so these settings here should be the ones relative to local development setup (through docker)

Comment (gunzip) [reply README.md]: can you add the settings to setup spid validator integration ? (local dev only, to let developers login using docker-compose and verify spid validator constraints)

Comment (BurnedMarshal) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -249,6 +253,43 @@ Kubernetes secrets:
...
+|----------------------------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------|
+| `IDP_METADATA_URL`                     | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml            | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml |
+| `CIE_METADATA_URL`                     | https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth              | https://idserver.servizicie.interno.gov.it/idp/shibboleth        |
+| `SPID_TESTENV_URL`                     | https://spid-testenv2:8088                                                  | -                                                                |
+
+On local development only spid-testenv2, executed with docker-compose, could be used to complete a SPID login.
+
+On production in order to reduce the startup time could be provided another Environment varible `STARTUP_IDPS_METADATA`. The command that must be used to set the variable is:
```
Comment (gunzip) [inline README.md]:
```suggestion
On production, in order to reduce the startup time, another Environment variable could be provided  `STARTUP_IDPS_METADATA`. To set its value run: 
```

File: README.md [hunk 4]
```diff
@@ -249,6 +253,43 @@ Kubernetes secrets:
...
+
+The default values are explained into the following table
+
+| ENV                                    | development                                                                 | production                                                       |
+|----------------------------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------|
+| `IDP_METADATA_URL`                     | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml            | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml |
+| `CIE_METADATA_URL`                     | https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth              | https://idserver.servizicie.interno.gov.it/idp/shibboleth        |
+| `SPID_TESTENV_URL`                     | https://spid-testenv2:8088                                                  | -                                                                |
+
+On local development only spid-testenv2, executed with docker-compose, could be used to complete a SPID login.
```
Comment (gunzip) [inline README.md]:
```suggestion
For local development only, spid-testenv2 (executed with docker-compose) could be used to login with a (fake) SPID account.
```

**Pull request merged**